
float32 angle_base
float32 angle_aruco
float32 angle_cam